### PR TITLE
Removed unnecessary refreshing from unrecognized series.

### PR DESCRIPTION
**Pull request by bond4u**

### CODE REVIEW

**Comment by da3dsoul** [comment]
Me: "ugh I'm sure it's fine."
Narrator: "It was not fine"

**Comment by da3dsoul** [comment]
Basically, it's fine to not refresh series with nothing selected, but when you select something, it sanitizes the name and searches it. When you select multiple, it searches the intersection.

File: Shoko.Desktop/UserControls/UnrecognisedVideos.xaml.cs [hunk 1]
```diff
@@ -214,7 +214,10 @@ private void DgVideos_SelectionChanged(object sender, SelectionChangedEventArgs
                 }
                 SetConfirmDetails();
                 Confirm1.Visibility = Confirm2.Visibility = visible;
-                if (string.IsNullOrEmpty(txtSeriesSearch.Text)) RefreshSeries();
+                if (string.IsNullOrEmpty(txtSeriesSearch.Text) && !isShiftOrCtrlDown) {
```
Comment (da3dsoul) [inline Shoko.Desktop/UserControls/UnrecognisedVideos.xaml.cs]: This is good

File: Shoko.Desktop/UserControls/UnrecognisedVideos.xaml.cs [hunk 2]
```diff
@@ -321,8 +324,6 @@ void btnAddSeries_Click(object sender, RoutedEventArgs e)
                 bool? result = frm.ShowDialog();
                 if (result.HasValue && result.Value)
                 {
-                    RefreshSeries();
```
Comment (da3dsoul) [inline Shoko.Desktop/UserControls/UnrecognisedVideos.xaml.cs]: Add a check. If there's something selected, then refresh

**Comment by da3dsoul** [comment]
Okay, check those and we shall talk

**Comment by bond4u** [comment]
can't get any better

**Comment by da3dsoul** [comment]
Looks good

**Pull request merged**